languages/ruby/lib/oso.rb

```diff
@@ -2,11 +2,14 @@
 
 require 'oso/oso'
 require 'oso/polar'
+require 'oso/errors'
 require 'oso/version'
 
 # Top-level namespace for oso authorization library.
 module Oso
   def self.new
+    # TODO: deprecate this method. Instead, users should use:
+    # > oso = Oso::Oso.new
```

Comment:
Why?

Reply:
Whoops, this TODO made more sense when we were changing to using `Enforcer`